source/agora/consensus/data/Params.d

```diff
@@ -96,14 +97,15 @@ public immutable class ConsensusParams
     /// Default for unittest, uses the test genesis block
     version (unittest) public this (
         uint validator_cycle = 1008, uint max_quorum_nodes = 7,
-        uint quorum_threshold = 80)
+        uint quorum_threshold = 80, Amount min_fee = 0)
     {
         import agora.consensus.data.genesis.Test : GenesisBlock;
         import agora.utils.WellKnownKeys;
         ConsensusConfig config = {
             validator_cycle: validator_cycle,
             max_quorum_nodes: max_quorum_nodes,
             quorum_threshold: quorum_threshold,
+            min_fee: min_fee,
```

Comment:
Whenever we have a new consensus parameter, while it's good to add it to the struct, we really don't need to make it configurable unless there's a reason to. ATM I don't really see one for this ?

Reply:
Our network tests doesn't have any fee, and the default minimum fee is 700, so I have to override it

Reply:
In our unit tests, we almost never use fees and the default fee is 700, so they would fail.